apps/widget/src/components/notification-center/NotificationCenterWidget.tsx

```diff
@@ -39,6 +39,7 @@ export function NotificationCenterWidget(props: INotificationCenterWidgetProps)
   const [colorScheme, setColorScheme] = useState<ColorScheme>('light');
   const [doLogout, setDoLogout] = useState(false);
   const [preferenceFilter, setPreferenceFilter] = useState<(userPreference: IUserPreferenceSettings) => boolean>();
+  const [showUserPreferences, setShowUserPreferences] = useState<boolean>(false);
```

Comment:
@ruhani2 I feel that true should be the default value. If user do not want it he can set it to false, but by default it will show.

Reply:
Ohh, that's correct. let me update it. Thanks for pointing that out.

Reply:
updated.